crates/bevy_pbr/src/light.rs

```diff
@@ -14,14 +14,16 @@ pub struct Light {
     pub color: Color,
     pub fov: f32,
     pub depth: Range<f32>,
+    pub attenuation: f32,
 }
 
 impl Default for Light {
     fn default() -> Self {
         Light {
-            color: Color::rgb(1.0, 1.0, 1.0),
+            color: Color::rgba(1.0, 1.0, 1.0, 100.0),
```

Comment:
Is the alpha supposed to be the light intensity? If so, why do it this way when we could have a separate `intensity` field and then let the LightNode handle combining it with the color before uploading it to the shader?

Reply:
It was indeed intensity and it's now a separate field on `Light`.